### ISSUE TITLE
I can't install it.

### ISSUE THREAD

**Issue by Immortal-Sty** [bneijt/autotrash#40]
System: Ubuntu 20.04 LTS (WSL);

I input :

```
sudo apt install -y autotrash
```

And it said :

```
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package autotrash is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'autotrash' has no installation candidate
```

**Comment by Immortal-Sty** [comment]
Thank you. I change "focal" to "bionic" in apt source. Then I get it.

> Seems like it's only available on bionic (18.04 LTS): https://packages.ubuntu.com/search?keywords=autotrash&searchon=names&suite=all&section=all. Not sure if there's a way to add that repo in focal (20.04 LTS), but in the meantime you can install it with instead.`pip`

**Comment by bneijt** [comment]
> Uhhh I'm not sure that's such a good idea since bionic is for Ubuntu 18.04 and focal is for 20.04 (so most things in bionic will be outdated for you and may cause dependency issues).

Agree, mixing Ubuntu releases is not a good thing. You are probably better of with just installing it via python-pip directly and adding `~/.local/bin` to your `PATH` session environment variable.

Glad to hear you got it working and thank you for reaching out and posting an issue.
